src/wp-includes/class-wp-theme-json.php

```diff
@@ -2901,9 +2901,9 @@ public function get_root_layout_rules( $selector, $block_metadata ) {
 			// Alignfull children of the container with left and right padding have negative margins so they can still be full width.
 			$css .= '.has-global-padding > .alignfull { margin-right: calc(var(--wp--style--root--padding-right) * -1); margin-left: calc(var(--wp--style--root--padding-left) * -1); }';
 			// Nested children of the container with left and right padding that are not wide or full aligned do not get padding.
```

Comment:
```suggestion
			// Nested children of the container with left and right padding that are not wide or full aligned do not get padding, unless they are direct children of an alignfull flow container.
```
The above suggestion is just the [comment update](https://github.com/WordPress/gutenberg/pull/62670/files#diff-b03597cc3da199e5aa5a94950e8241135904853e7c3b82d758e42744878afae7R2936) from the Gutenberg PR. Should we also update the `@since 6.6.0` comment to reflect the change?